types/helmet/index.d.ts

```diff
@@ -69,8 +69,8 @@ declare namespace helmet {
         objectSrc?: HelmetCspDirectiveValue[];
         pluginTypes?: HelmetCspDirectiveValue[];
         prefetchSrc?: HelmetCspDirectiveValue[];
-        reportTo?: string;
-        reportUri?: string;
+        reportTo?: HelmetCspDirectiveValue | false;
```

Comment:
The `| false` is a bit odd but it does seem to match the documentation. Since leaving it empty is the same as setting it to `false`, it might be worth omitting this, though it's probably up to the Helmet maintainers who I believe are tagged in this.

Reply:
I'm thinking of removing this in the next version of Helmet, so I think it's safer to remove the `| false` bit. As @BlueHatbRit said, it _does_ match the docs, but because I'm thinking about removing it, I'd omit it.

Reply:
Sure, I will remove it